src/components/Footer/styles/_35-footer-links.scss

```diff
@@ -21,6 +21,7 @@
   margin: 0 var(--space-unit);
   position: relative;
   text-decoration: none;
+  @include animated-underline;
```

Comment:
put the `@include` at the top of the style block, after any `@extend`s

Reply:
done. re-review pls.

Reply:
This should be caught by the style-lint, I'll investigate

Reply:
I did do a lint checking, there was no error.

Reply:
yep I know - but there _should_ be an error 😬 writing some new lint rules now https://github.com/hudochenkov/stylelint-order/blob/master/rules/order/README.md